src/dodal/devices/utils.py

```diff
@@ -12,3 +18,71 @@ def epics_signal_put_wait(pv_name: str, wait: float = 1.0) -> EpicsSignal:
         EpicsSignal: An EpicsSignal that will wait for a callback.
     """
     return Component(EpicsSignal, pv_name, put_complete=True, write_timeout=wait)
+
+
+def wrap_and_do_funcs(unwrapped_funcs, timeout=60):
```

Comment:
Type annotations strongly recommended here, makes IDE code completion much nicer
```suggestion
def wrap_and_do_funcs(
    unwrapped_funcs: list[Callable[[], Status]],
    timeout: float = 60.0,
) -> Status:
```